LondonBritishLibrary/orient/BLorient691.xml

```diff
@@ -31,21 +31,206 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+                      (<date ref="ethiocal:Hedar24">24th of Ḫǝdār</date>)</title>
+                    <incipit>አመ፡ ፳ወ፬፡ ለኅዳር፡ ተግሣጽ፡ ለካሕናት፡ ዘደረሰ፡ አብ፡ ቅዱስ፡ አባ፡ ሳዊሮስ፡ ኤጲስ፡ ቆጶስ፡ ዘሰሜኖን፡ ጸሓፊ፡ ግብጻዊ፡ አመ፡ ፳ወ፬፡ ለኅዳር፡
+                    በሰላመ፡ እግዚአብሔር፡ አሜን።</incipit>
+                    <textLang mainLang="gez"/>
+                    </msItem>
+
+                    <msItem xml:id="ms_i14">
+                    <locus from="199r"/>
+                    <title xml:lang="en" type="complete" ref="LIT1899Martyr">Martyrdom of Mercurius (<date ref="ethiocal:Hedar25">25th of Ḫǝdār</date>)</title>
+                    <textLang mainLang="gez"/>
+                    </msItem>
+
+                    <msItem xml:id="ms_i15">
+                    <title type="complete" xml:lang="en" ref="LIT6173JamesIn">Jacob the Persian, who was cut in pieces (<date ref="ethiocal:Hedar27">27th of Ḫǝdār</date>)</title>
+                    <incipit>አመ፡ ፳ወ፯፡ ለኅዳር፡ ስምዕ፡ ዘቅዱስ፡ ግሙድ፡ መስተጋድል፡ ያዕቆብ፡ ዘረከቦ፡ ዘተፈጸመ፡ ስምዑ፡ በምድረ፡ ፋርስ፡ በሀገር፡ እንተ፡ ስማ፡
+                    በልቃ፡ በመንግሥተ፡ ፋርስ፡ በመዋዕለ፡ ሰርደ፡ ንጉሥ፡ ወውእቱሰ፡ ክርስቲያናዊ፡ ያአምን፡ በክርስቶስ፡ በሰላመ፡ እግዚአብሔር፡ አሜን።</incipit>
+                    <textLang mainLang="gez"/>
+                    </msItem>
+
+                    <msItem xml:id="ms_i16">
+                    <locus from="219r"/>
+                    <title type="complete" xml:lang="en">Discourse on S. Matthew, ch. 10, 37-39</title>
+                    <textLang mainLang="gez"/>
+                    </msItem>
+
+                    <msItem xml:id="ms_i17">
+                    <locus from="223r"/>
+                    <title type="complete" xml:lang="en">Discourse on S. Luke, ch. 9, 23-25</title>
+                    <textLang mainLang="gez"/>
+                    </msItem>
+
+                    <msItem xml:id="ms_i18">
+                    <locus from="224r"/>
+                    <title type="complete" xml:lang="en" ref="LIT1895Martyr">Martyrdom of Eustathius</title>
+                    <textLang mainLang="gez"/>
+                    <note>Imperfect at the end.</note>
+                    </msItem>
+
+                    <msItem xml:id="ms_i19">
+                    <locus from="228r"/>
+                    <title type="complete" xml:lang="en" ref="LIT1895Martyr">Martyrdom of Eustathius, his wife and two children, under Marcianus (<date ref="ethiocal:Maskaram23">23rd of Maskaram</date>)</title>
+                    <textLang mainLang="gez"/>
+                    </msItem>
+
+                    <msItem xml:id="ms_i20">
+                    <colophon xml:id="coloph1" xml:lang="gez">
+                    መልአ፡ መጽሐፈ፡ እንባእቆም፡ ነቢይ፡ ዐቢይ፡ አመ፡ ፲ወ፰፡ ለጥቅምት፡ በ<date calendar="grace" when="0477">፬፻፸፯: ዓመተ፡ ምሕረት፡</date> በመዋዕለ፡ <persName ref="PRS10215Yagbas">ያግባ፡<sic/> ጽዮን፡</persName> ንጉሥ፡ መፍቀሬ፡
+                    እግዚአብሔር፡ ዘተሠርዐ፡ በመዋዕሊሁ፡ ከመ፡ ይትአዘዙ፡ ሕዝበ፡ ኢትዮጵያ፡፡ ከመ፡ ይትዓገሱ፡ ከመ፡ ክርስቲያን፡ በከ፡ <gap reason="ellipsis" resp="PRS10171Wright"/>
+                    <note>The note is written by a later hand.</note>
```

Comment:
I wonder if this "colophon" should rather be reagarded as an addition, as it is written in a later hand and has little to do with a real colophon (the dating is rather a fantasy). What do you think @thea-m ?

Reply:
Yes, if it is added by a later hand I would also encode it as an addition.